### PR TITLE
Patch 0.1.3

### LINKED ISSUE #10: Make absolute positioned element do not impact layout size when it is autosized basing on content size

### LINKED ISSUE #11: implement remove by id method

### LINKED ISSUE #12: Add stories that shows dynamic add/remove content to the layout

### LINKED ISSUE #13: Add stories that shows how to change styles of some inner layout

### LINKED ISSUE #14: Resize all the layout tree on some element style change

### LINKED ISSUE #15: Remove text style nesting for children

### PR DESCRIPTION
**Pull request by CyberDex**
Closes #10 
Closes  #11
Closes #12
Closes #13
Closes #14 
Closes #15

### CODE REVIEW

File: src/controllers/ContentController.ts [hunk 1]
```diff
@@ -37,93 +50,151 @@ export class ContentController
...
                 }
+                break;
+            case 'array':
+                const contentArray = content as Array<LayoutOptions>;
 
-                const newLayout = new Layout(contentConfig);
-
-                this.children.set(newLayout.id, newLayout);
-                this.layout.addChild(newLayout);
+                contentArray.forEach((content) => this.createContent(content, parentGlobalStyles));
```
Comment (CyberDex) [inline src/controllers/ContentController.ts:175]: This is just a refactoring with few fixes.

File: src/controllers/SizeController.ts [hunk 2]
```diff
@@ -127,6 +127,11 @@ export class SizeController
 
                         if (child instanceof Layout && child.style.display !== 'block')
                         {
+                            if (child.style.position)
+                            {
+                                return;
+                            }
+
```
Comment (CyberDex) [inline src/controllers/SizeController.ts:134]: Absolute positioned elements should not impact autosize of the parent layout

File: src/controllers/SizeController.ts [hunk 3]
```diff
@@ -138,7 +143,7 @@ export class SizeController
                     // height is basing on content height
                     finalWidth = childrenWidth + paddingLeft + paddingRight;
 
-                    if (this.isItJustAText)
+                    if (isItJustAText(this.layout))
```
Comment (CyberDex) [inline src/controllers/SizeController.ts:146]: Just a abstraction of the method to helpers.

File: src/controllers/SizeController.ts [hunk 4]
```diff
@@ -198,7 +203,10 @@ export class SizeController
                     // add first element as at lease one element to set width
                     if (firstChild instanceof Layout)
                     {
-                        childrenHeight += firstChild.height;
+                        if (!firstChild.style.position)
+                        {
+                            childrenHeight += firstChild.height;
+                        }
```
Comment (CyberDex) [inline src/controllers/SizeController.ts:209]: Absolute positioned elements should not impact autosize of the parent layout

File: src/controllers/StyleController.ts [hunk 5]
```diff
@@ -1,230 +1,18 @@
...
-    TextStyleFontVariant,
-    TextStyleFontWeight,
-    TextStyleLineJoin,
-    TextStyleTextBaseline,
-    TextStyleWhiteSpace,
-} from '@pixi/text';
-import { TEXT_GRADIENT, TextStyle } from '@pixi/text';
-import { getColor } from '../utils/helpers';
-import { OVERFLOW, ALIGN, VERTICAL_ALIGN } from '../utils/constants';
+import type { GradeToOne, Styles } from '../utils/types';
```
Comment (CyberDex) [inline src/controllers/StyleController.ts:1]: This controller is refactored a lot. Same logic but with aiming to more easy control over the properties. 

All the control now happens through the `set` method.

**Review by Zyie — APPROVED** [review]

**Pull request merged**